docs/topics/db/models.txt

```diff
@@ -1609,3 +1609,38 @@ and keeping code analysis tools useful.
     :doc:`The Models Reference </ref/models/index>`
         Covers all the model related APIs including model fields, related
         objects, and ``QuerySet``.
+
+Using through models with ManyToManyField
+=========================================
+
+By default, when Django creates an implicit through table for a ``ManyToManyField``,
+it automatically creates a unique index on the pair of foreign keys. For example::
+
+    class Author(models.Model):
+        name = models.CharField(max_length=100)
+
+
+    class Book(models.Model):
+        title = models.CharField(max_length=100)
+        authors = models.ManyToManyField(Author)
+
+In this case, Django automatically creates a unique index for the through table.
+
+When using a custom through model, you need to explicitly define any unique
```

Comment:
Can you also add the example of custom through model for manytomanyfield as well? From my knowledge I can not see that. or what am I missing?